frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/BaseSlideOutBranchAction.java

```diff
@@ -54,18 +49,18 @@ public void doSlideOut(AnActionEvent anActionEvent, BaseGitMacheteNonRootBranch
     String branchName = branchToSlideOut.getName();
     Project project = ActionUtils.getProject(anActionEvent);
     var branchLayout = ActionUtils.getBranchLayout(anActionEvent);
-    var gitMacheteFilePath = ActionUtils.getGitMacheteFilePath(anActionEvent);
-    if (branchLayout.isEmpty() || gitMacheteFilePath.isEmpty()) {
+    var branchLayoutManager = ActionUtils.getBranchLayoutManager(anActionEvent);
```

Comment:
Hmmm it seems that we don't need the access to the reader via the `DataKey`, just to the writer... maybe the `DataKey` should then just point to the writer, not the entire manager then?